pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     "packaging >=21.3,<24.0.0",
     "jinja2 >=3.1.2,<4.0.0",
     "SQLAlchemy >=2.0.0,<3.0.0",
+    "defusedxml >=0.7.1,<1.0.0"
```

Comment:
Please add `,` at the end of the line. That helps keeping git diff lines relevant in future.